### ISSUE TITLE
'and' thrown error

### ISSUE THREAD

**Issue by gregtatum** [tmpvar/2d-polygon-boolean#14]
Is [this line of code](https://github.com/tmpvar/2d-polygon-boolean/blob/master/2d-polygon-boolean.js#L276) just saying that no results were found? If so I would prefer just returning empty rather than throwing an error. Not 100% sure if I'm interpreting it correctly.

**Comment by tmpvar** [comment]
yeah, I think you've got that right.  This package is about to go through a major overhaul that removes this code and fixes other numerical robustness issues / degenerate cases.

I'll work on getting this refactor done tonight.

**Comment by o0starshine0o** [comment]
it's still throwing this error, have you fixed this issues?
